apps/base-docs/docs/tools/node-providers.md

```diff
@@ -62,6 +62,17 @@ hide_table_of_contents: true
 
 ---
 
+## All That Node
+
+[All That Node](https://www.allthatnode.com/base.dsrv) is the most comprehensive multi-chain development suite, designed to support multiple networks from a single platform. They offer free and [paid plans](https://www.allthatnode.com/pricing.dsrv) with additional features and increased request limits.
```

Comment:
@yunseonna 

Could you also please update to remove the superlative?

s/is the most/is a/